code/modules/power/antimatter/control.dm

```diff
@@ -108,14 +110,16 @@
 				toggle_power()
 			stability -= rand(10,20)
 	..()
-	return 0
+	return FALSE
 
 
 /obj/machinery/power/am_control_unit/blob_act()
 	stability -= 20
-	if(prob(100-stability))//Might infect the rest of the machine
+	if(prob(100-stability))//Might infect the rest of the machine! Watch out!
+		for(var/obj/machinery/am_shielding/AMS in linked_cores)
+			AMS.blob_act(2)

```

Comment:
what exactly is the number doing here? blob_act is expecting /obj/structure/blob as the arg